### ISSUE TITLE
HoloISO is now immutable  - are repos dead? What happens next? 

### ISSUE THREAD

**Issue by iamhumanipromise** [zweiler2/HoloISO_Nvidia#22]
Are the repos going to be OK? Or for now just hold on until there is an immutable project from main HoloISO? *I'm so confused*

**Comment by mucookul** [comment]
The Main HoloISO is now immutable, so I guess use that. https://github.com/holoiso-staging/releases is official

**Comment by zweiler2** [comment]
@iamhumanipromise Nothings dead. The repos are fine. I'll probably stay mutable, not 100% decided yet tough.

If you want immutability(and want better update system, than downloading my ISO for every release to get my changes), you can switch to upstream easily with their migration tool (which is included in V1.1). The link @Mukul1127 provided is exactly where you should go.

As to what happens next:
As upstream has now support for nvidia and is using a much better update system (tough immutable), i don't see much of a reason for this fork anymore. We'll see. But for the near future, i'll continue my work.

**Comment by QuillFlash** [comment]
In my opinion, choice is very important within the Linux ecosystem. My feedback is to rename the project to “HoloISO Unlocked” or something else that represents both the roots with HoloISO and the main gimmick of the distro: that this is basically like SteamOS, but with an unlocked root partition.

Then, you should continue working on the project, maybe even making pre-compiled packages for upstream’s better Nvidia support now and publish the tweaks in your own repo.

TL;DR: I’d love you to keep the project and market it as a “mutable” SteamOS alternative. I genuinely like the tinkering aspect of an unlocked file system, so you consider providing that choice for the people.

**Comment by QuillFlash** [comment]
@zweiler2 In my opinion, choice is very important within the Linux ecosystem. My feedback is to rename the project to “HoloISO Unlocked” or something else that represents both the roots with HoloISO and the main gimmick of the distro: that this is basically like SteamOS, but with an unlocked root partition.

Then, you should continue working on the project, maybe even making pre-compiled packages for upstream’s better Nvidia support now and publish the tweaks in your own repo.

TL;DR: I’d love you to keep the project and market it as a “mutable” SteamOS alternative. I genuinely like the tinkering aspect of an unlocked file system, so you consider providing that choice for the people.

**Comment by QuillFlash** [comment]
@zweiler2 In my opinion, choice is very important within the Linux ecosystem. My feedback is to rename the project to “HoloISO Unlocked”, “PlumberISO” or something else that represents both the roots with HoloISO and the main gimmick of the distro: that this is basically like SteamOS, but with an unlocked root partition.

Then, you should continue working on the project, maybe even making pre-compiled packages for upstream’s better Nvidia support now and publish the tweaks in your own repo.

TL;DR: I’d love you to keep the project and market it as a “mutable” SteamOS alternative. I genuinely like the tinkering aspect of an unlocked file system and the infinite possibilites of available packages in the AUR, so you should consider providing that choice for the people.

**Comment by zweiler2** [comment]
Thank you for your input. Staying mutable makes sense i think, especially with the trend of many SteamOS projects going immutable. As for the name, i think it still fits cause you can tweak the system the way you like it. As for maintaining a package repo, i'll have a chat with @Ew4n1011 (my server maintainer) in private for what/how/when is possible.

I never said upstream has better nvidia support (saw (i assume) your post on telegram). Just that they have it now and before they went immutable did not (which was one of the reasons i started the fork).
I said the update system is better, especially if you don't want to tinker with the system.
Oh and upstream source code is available.

I think Nvidia support on my fork is better (with the latest release), as upstream has no gamemode video acceleration yet(it's quite laggy).

I tried to write on telegram, but i'm blocked to write in public groups... 
Till 30th jan, then 3rd feb and now 5th feb...
Telegram bot says cause of looking in profiles in the public group, which i don't do... Hopefully it goes back to normal soon.

**Comment by QuillFlash** [comment]
@zweiler2 Thank you so much for clarification.

This means the only thing needed is to find a new name that reflects this is a mutable distro with an unlocked root partition in the sea of immutable SteamOS projects.

Having a mutable option is incredibly useful, as this combines the gaming prowess of Valve with the flexibility of the Arch User Repository and its tens of thousands of packages.

**Comment by zweiler2** [comment]
I don't think a new name is needed as it's exactly that; my tweaked version of HoloISO.
So i'll stick with that.
